pkgs/os-specific/linux/firmware/sof-firmware/default.nix

```
@@ -3,29 +3,28 @@
 with lib;
 stdenv.mkDerivation rec {
   pname = "sof-firmware";
-  version = "1.6";
+  version = "1.7";
 
   src = fetchFromGitHub {
     owner = "thesofproject";
     repo = "sof-bin";
-    rev = "cbdec6963b2c2d58b0080955d3c11b96ff4c92f0";
-    sha256 = "0la2pw1zpv50cywiqcfb00cxqvjc73drxwjchyzi54l508817nxh";
+    rev = "v${version}";
+    sha256 = "sha256-Z0Z4HLsIIuW8E1kFNhAECmzj1HkJVfbEw13B8V7PZLk=";
   };
 
   phases = [ "unpackPhase" "installPhase" ];
```

Please remove the phases overwrites as they tend to break hooks and are hard to debug.

---

Would this be acceptable?
```suggestion
  dontFixup = true;
```

---

That would work plus a comment with that info wouldn't hurt so no one asks you if the binaries can be stripped.